PowerShell/ModuleChangelog.md

```diff
@@ -1,3 +1,17 @@
+## 2.8.0
+
+Release Date: August 1, 2023
```

Comment:
Change release date to current or whenever we merge?

Reply:
Ah sorry I saw this just now yes we'll plan to do that!